vipps-checkout-api.md

```diff
@@ -1,52 +1,115 @@
-    - [Partner signup API guidelines](#partner-signup-api-guidelines)
-  - [System information guidelines](#system-information-guidelines)
-  - [Transaction operations (Capture, Cancel, Refund)](#transaction-operations-capture-cancel-refund)
-  - [Polling integration](#polling-integration)
-    - [Example of polling response when checkout SessionState is SessionStarted](#example-of-polling-response-when-checkout-sessionstate-is-sessionstarted)
-    - [Example of polling response when checkout SessionState is any other state but SessionStarted](#example-of-polling-response-when-checkout-sessionstate-is-any-other-state-but-sessionstarted)
-  - [Callback handling](#callback-handling)
-    - [Example callback request](#example-callback-request)
-  - [Shipping](#shipping)
-    - [Static Shipping](#static-shipping)
-    - [Dynamic Shipping](#dynamic-shipping)
-  - [Vipps side Transaction information](#vipps-side-transaction-information)
-    - [Recommended integration (currently in pilot mode)](#recommended-integration-currently-in-pilot-mode)
-  - [In-App Payment](#in-app-payment)
+  - [Flow diagram](#flow-diagram)
+  - [Step 1: Initiating a session](#step-1-initiating-a-session)
+    - [Configuration for use inside a native mobile application](#configuration-for-use-inside-a-native-mobile-application)
+  - [Step 2: Displaying the session to the user](#step-2-displaying-the-session)
+    - [Sticky checkout session](#sticky-checkout-session)
+  - [Step 3: Handling the result of the session](#step-3-handling-the-result-of-the-session)
+    - [Callback handling](#callback-handling)
+    - [Session polling](#session-polling)
+- [Integration partner and plugin guidelines](#integration-partner-and-plugin-guidelines)
+  - [Partner signup API guidelines](#partner-signup-api-guidelines)
+- [Transaction operations (Capture, Cancel, Refund)](#transaction-operations-capture-cancel-refund)
+- [Vipps side Transaction information](#vipps-side-transaction-information)
+  - [Recommended integration (currently in pilot mode)](#recommended-integration-currently-in-pilot-mode)
+
+# Checkout Features
+
+Explains the high level features of Vipps Checkout.
+
+## Vipps Checkout API
+
+Vipps Checkout works around the concept of a _session_, which has a time to live of one hour. The API exposes endpoints for the merchant to interract with a session. These include:
+
+- session initiation
+- session status
+- session cancellation
+
+## Vipps Checkout frontend
+
+Once a session is created, it is to be opened inside an iFrame embedded on the merchant website. The iFrame loads a web application that fetches all neccessary information about the session from Vipps.
+
+### SDK
+
+Vipps exposes an SDK to make opening the session on the merchant easy. It is highly recommended to use this.
+
+## Shipping
+
+In most situations a merchant wants to send good to a customer using a shipping provider. Vipp Checkout supports two different types of shipping configuration:
```

Comment:
VippS Checkout

Reply:
In most situations, a merchant wants to send goods   .....  
                            ^                                                    ^ 
Vipps Checkout supports two different types of shipping configurations
                                                                                                                  ^